extensions/integration-tests/src/schemaCompare.test.ts

```diff
@@ -238,7 +238,6 @@ class SchemaCompareTester {
 				});
 			}
 			assert(tasks !== null && tasks.tasks.length > 0, 'Tasks should still show in list. This is to ensure that the tasks actually complete.');
```

Comment:
Please add a //todo comment to say we need to add some proper validation for task completion later so that we don't forget